ui/app/styles/components/console-ui-panel.scss

```diff
@@ -39,7 +39,7 @@
   font-size: $body-size;
   font-weight: $font-weight-semibold;
   justify-content: flex-end;
-  min-height: 100%;
+  min-height: calc(100% - 35px); // account for close button that is sticky positioned
```

Comment:
This was causing the console panel to always overflow

Reply:
tip: convert `35px` to a Sass variable, so it can be shared between the close button and here, and you have a single point where to declare it (they will always be in sync in this way)